Makefile

```diff
@@ -94,4 +111,8 @@ help:
 	@echo '  the charm will be deployed in the default Juju environment.'
 	@echo '  If SERIES is not passed, "trusty" is used. Possible values are'
 	@echo -e '  "precise" and "trusty".\n'
+	@echo 'make publish-development - Push and publish the charm to the'
+	@echo -e '  development channel.\n'
+	@echo 'make publish-stable - Push and publish the charm to the'
+	@echo -e '  stable channel.\n'
```

Comment:
I'd rather remove publishing to stable through make script. This should always be manual.